src/Project.md

```diff
@@ -0,0 +1,10 @@
+# Project Documentation
+This project aims to build a webextension for Firefox browser, that can tag other Web extension in the browser.
```

Comment:
what does, "that can tag other web-extension" mean ?  you can either use the word "track" or any appropriate word in your convenience

Reply:
Actually I have copied this from the README.md file. But I will update it.

Reply:
So sorry, its my mistake then. I have fixed the typo in the readme. Thanks for pointing it out